### PR TITLE
v-e-c: print debug logging at bottom of output

### PR DESCRIPTION
**Pull request by mbestavros**
Ref: https://issues.redhat.com/browse/EC-565

Simple change to the `verify-enterprise-contract` Task that should capture debug output in a file, then print it at the bottom of the output.

This hasn't been tested, so I'm submitting it as a draft.

cc @zregvart

### CODE REVIEW

**Comment by lcarva** [comment]
I believe @zregvart is addressing this in https://github.com/enterprise-contract/ec-cli/pull/1769

**Comment by zregvart** [comment]
Yeah, I kinda lumped a bunch of issues into #1769, [`27e871a` (#1769)](https://github.com/enterprise-contract/ec-cli/pull/1769/commits/27e871a628d5da1f7866d282ac8a74ddd4b94a47) adds `--logfile` and [`751f2e5` (#1769)](https://github.com/enterprise-contract/ec-cli/pull/1769/commits/751f2e58ca3c5b9d02d9ff2504003d724cd960a0) uses it to output the debug logs at the bottom of the Task logs

File: tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml [hunk 1]
```diff
@@ -187,6 +187,8 @@ spec:
         - "appstudio=$(results.TEST_OUTPUT.path)"
         - "--output"
         - "json=$(params.HOMEDIR)/report-json.json"
+        - "2>"
+        - "debug.log"
```
Comment (zregvart) [inline tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml:191]: We can't use shell redirects here, these are arguments passed to `ec` directly, not a shell invocation...

**Comment by lcarva** [comment]
Done in https://github.com/enterprise-contract/ec-cli/pull/1769. Let's close it.

**Pull request closed without merging**